assets/js/blocks/active-filters/block.tsx

```diff
@@ -34,7 +33,7 @@ import {
 } from './utils';
 import ActiveAttributeFilters from './active-attribute-filters';
 import FilterPlaceholders from './filter-placeholders';
-import { Attributes } from './types';
+import { ActiveFiltersBlockProps } from './types';
```

Comment:
```suggestion
import type { ActiveFiltersBlockProps } from './types';
```

Using `import type { ActiveFiltersBlockProps } from './types';` instead of `import { ActiveFiltersBlockProps } from './types';` leads to a faster compile time. I'd say we should do that in this case, given that `ActiveFiltersBlockProps` is a type that we're importing.

Reply:
For sure! thanks for the explanation this is valuable information.